liblangutil/Exceptions.h

```diff
@@ -119,7 +119,9 @@ class Error: virtual public util::Exception
 		ParserError,
 		TypeError,
 		SyntaxError,
-		Warning
+		GeneralError,
```

Comment:
Why not move this to the beginning and just call it `Error`?

Reply:
Fine with me